### ISSUE TITLE
Deprecate Node.js <= 4 and support v6

### ISSUE THREAD

**Issue by ilyazub** [twolfson/canvassmith#6]
Currently Travis CI runs builds on deprecated versions of Node.js. Let's remove [v0.x](https://github.com/twolfson/canvassmith/blob/aafcdaaddda19d1f48f99bd837f1800f1c454c7e/.travis.yml#L4-L6) and add v6 to `.travis.yml`.

**Comment by twolfson** [comment]
I'm fully open to this. I typically only update these when I haven't touched a repo in a while.

I'll take care of this over the next few days.

For personal reference, this `.travis.yml` should look like

```yml
sudo: false
language: node_js
node_js:
  - "7"
  - "6"
  - "4"
matrix:
  allow_failures:
    - node_js: "7"

before_install:
  - curl --location http://rawgit.com/twolfson/fix-travis-ci/master/lib/install.sh | bash -s

notifications:
  email:
    recipients:
      - todd@twolfson.com
    on_success: change
    on_failure: change
```

**Comment by twolfson** [comment]
This has been resolved in `1.2.0`. Thanks for the issue!
